src/js/view/dock.js

```diff
@@ -1,365 +1,45 @@
 /*jshint evil:true*/
 define([
-    'view/overlay',
-    'view/touch',
-    'events/events',
+    'handlebars-loader!templates/dock.html',
     'utils/helpers',
-    'utils/css',
-    'underscore'
-], function(Overlay, Touch, events, utils, cssUtils, _) {
+    'utils/underscore'
```

Comment:
...fixed by removing alias so that we don't have issues or conflicts between requirejs and webpack https://github.com/jwplayer/jwplayer/commit/295b739eec48927239aa56b915e1f8f0df5d69a7

Looks like require wasn't event trying to load 'utils/underscore' because it already loaded 'underscore' which was setup to load from the same path in require's config.